app/main.ipynb

```diff
@@ -176,82 +263,135 @@
     "    [1, 7]\n",
     "])"
    ],
-   "metadata": {
-    "collapsed": false,
-    "is_executing": true
-   }
+   "outputs": [],
+   "execution_count": 20
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
    "source": [
     "### Task 5.1 Find the mean for each column by `x` axis"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
    "metadata": {
     "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
     "ExecuteTime": {
-     "end_time": "2023-07-03T13:19:27.412040Z",
-     "start_time": "2023-07-03T13:19:27.406096Z"
+     "end_time": "2024-12-27T17:41:32.960277Z",
+     "start_time": "2024-12-27T17:41:32.939948Z"
     }
-   }
+   },
+   "source": "A.sum(axis=1)",
```

Comment:
In Task 5.1, you are calculating the sum of each row with `A.sum(axis=1)`. To find the mean for each column, you should use `A.mean(axis=0)`.